ext/alias_editor/main.php

```diff
@@ -91,7 +136,17 @@ public function onPageRequest(PageRequestEvent $event)
                 $t->token = $user->get_auth_token();
                 $t->inputs = $_GET;
                 $t->size = $config->get_int('alias_items_per_page', 30);
+				$input = validate_input(["r_oldtag"=>"string,optional", "r_newtag"=>"string,optional"]);
+				$tag_inputs = array();
+				if (isset($_GET["r_oldtag"])) {
+					$tag_inputs["r_oldtag"] = $_GET["r_oldtag"];
+				}
+				if (isset($_GET["r_newtag"])) {
+					$tag_inputs["r_newtag"] = $_GET["r_newtag"];
+				}
+				$t->inputs = array_merge($t->inputs, $input, $tag_inputs);
```

Comment:
This looks like a complicated way of taking inputs from both GET and POST at the same time? But it does so using 3 different mechanisms (and it's not clear why anything beyond the default GET support is needed). If we do actually need to support both GET and POST I'd do it with `$t->inputs = array_merge($_GET, $_POST)`

Reply:
The search button and the pagination uses different mechanisms.  If I don't do it this way, then you can search for a tag but if you try to go to page 2, the search fields will not persist.